Tools/LyTestTools/ly_test_tools/_internal/managers/workspace.py

```diff
@@ -65,7 +65,7 @@ def __init__(self,  # type: AbstractWorkspaceManager
         if not self.output_path:
             self.output_path = os.path.join(self.paths.test_results(),
                                             datetime.datetime.now().strftime('%Y-%m-%dT%H_%M_%S_%f'))
-            logger.info(f"No logs path set, using default based on timestamp: {self.output_path}")
```

Comment:
Unless another context is already logging this, I think this path is important enough to log as info, possibly even as a warning